Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -1127,28 +1089,28 @@ Note: To enrich an incident created via the Demisto BOT (`new incident` command)
 
     This probably means that there is a connection issue, and the web server does not intercept the HTTPS queries from Microsoft Teams.
 
-    In order to troubleshoot, first verify the Docker container is up and running and publish the configured port to the outside world:
-
-    From the Cortex XSOAR / Cortex XSOAR engine machine run: `docker ps | grep teams`
+    Steps o troubleshoot:
+   1. first verify the Docker container is up and running and publish the configured port to the outside world (currently not supported in XSOAR 8):
 
-    You should see the following, assuming port 7000 is used:
+       From the Cortex XSOAR / Cortex XSOAR engine machine run: `docker ps | grep teams`
 
-    `988fdf341127        demisto/teams:1.0.0.6483      "python /tmp/pyrunne…"   6 seconds ago       Up 4 seconds        0.0.0.0:7000->7000/tcp   demistoserver_pyexecLongRunning-b60c04f9-754e-4b68-87ed-8f8113419fdb-demistoteams1.0.0.6483--26`
+       You should see the following, assuming port 7000 is used:
 
-    If the Docker container is up and running, try running cURL queries, to verify the web server is up and running and listens on the configured URL:
+       `988fdf341127        demisto/teams:1.0.0.6483      "python /tmp/pyrunne…"   6 seconds ago       Up 4 seconds        0.0.0.0:7000->7000/tcp   demistoserver_pyexecLongRunning-b60c04f9-754e-4b68-87ed-8f8113419fdb-demistoteams1.0.0.6483--26`
 
-     - To the messaging endpoint from a separate box.
-     - From the Cortex XSOAR machine to localhost.
+       If the Docker container is up and running, try running cURL queries, to verify the web server is up and running and listens on the configured URL:
 
-       - Note: The web server supports only POST method queries.
+        - To the messaging endpoint from a separate box.
+        - From the Cortex XSOAR machine to localhost.
 
-    If the cURL queries were sent successfully, you should see in Cortex XSOAR logs the following line: `Finished processing Microsoft Teams activity successfully`.
+          - Note: The web server supports only POST method queries. 
+   2. If the cURL queries were sent successfully, you should see in Cortex XSOAR logs the following line: `Finished processing Microsoft Teams activity successfully`.
 
-    If you're working with secured communication (HTTPS), make sure that you provided a valid certificate, run `openssl s_client -connect <domain.com>:443` command, verify that the returned value of the `Verify return code` field is `0 (ok)`, otherwise, it's not a valid certificate.
+   3. If you're working with secured communication (HTTPS), make sure that you provided a valid certificate, run `openssl s_client -connect <domain.com>:443` command, verify that the returned value of the `Verify return code` field is `0 (ok)`, otherwise, it's not a valid certificate.
```

Comment:
```suggestion
   3. If you're working with secured communication (HTTPS), make sure that you provided a valid certificate.
       1. Run `openssl s_client -connect <domain.com>:443` .
       2. Verify that the returned value of the `Verify return code` field is `0 (ok)`, otherwise, it's not a valid certificate.
```